.bazelversion

```diff
@@ -1 +1 @@
-7.1.1
+8.3.1
```

Comment:
Are we also upgrading Bazel?

Reply:
yes, but this is valid when running targets in this module only.
It doesn't affect the bazel version used by repos that import this